### PR TITLE
Feature/bocm 287 get appeals for inspector view

### PR DESCRIPTION
**Pull request by mayya-bond-kc**
## Describe your changes

Creates endpoint for inspector to get list of appeals assigned to them

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [x] I have made corresponding changes to the documentation
- [x] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [x] I have provided a description of the ticket
- [x] I have included unit tests to cover any testable code changes

### CODE REVIEW

**Review by MariaValeroGonzalezKC — APPROVED** [review]

File: apps/api/src/server/app/utils/address-formatter-lowercase.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+const formatAddressLowerCase = function(address) {
+	return {
+		addressLine1: address.addressLine1,
+		addressLine2: address.addressLine2,
+		town: address.town,
+		county: address.county,
+		postCode: address.postcode
```
Comment (brpins) [inline apps/api/src/server/app/utils/address-formatter-lowercase.js:7]: FYI, "postcode" is a single word and so having this function to change its case isn't really necessary...

**Pull request merged**